system/jlib/jstring.cpp

```diff
@@ -2761,3 +2761,33 @@ StringBuffer& StringBuffer::operator=(StringBuffer&& value)
     swapWith(value);
     return *this;
 }
+
+
+bool loadBinaryFile(StringBuffer & contents, const char *filename, bool throwOnError)
+{
+    int fd = open(filename, O_RDONLY);
+    bool ok = false;
+    if (fd != -1)
+    {
+        const unsigned chunkSize = 0x10000;
+        ssize_t bytes;
+        for (;;)
+        {
+            void * buffer = contents.reserve(chunkSize);
+            bytes = (size_t)read(fd, buffer, chunkSize);
+            if (bytes != chunkSize)
+                break;
+        }
+
+        if (bytes >= 0)
+            ok = true;
```

Comment:
May be fine, but if bytes was 0 first time should this still return true ?
Perhaps change return at end to also check for contents.length() == 0 is contents.isEmpty() ?
I mean - should not ever happen but just in case.